tests/Pagination/PaginationPaginatorTest.php

```diff
@@ -174,4 +174,11 @@ public function testSimplePaginatorReturnsRelevantContextInformation()
             'prev_page_url' => '/?page=1', 'from' => 3, 'to' => 4, 'data' => ['item3', 'item4'],
         ], $p->toArray());
     }
+
+    public function testPaginatorGeneratesUrlsWithoutTrailingSlash()
```

Comment:
@alexrussell the existing tests do that
On Tue, 8 Sep 2015 at 16:14 Alex Russell <notifications@github.com> wrote:

> In tests/Pagination/PaginationPaginatorTest.php
> <https://github.com/laravel/framework/pull/10186#discussion_r38929967>:
>
> > @@ -174,4 +174,11 @@ public function testSimplePaginatorReturnsRelevantContextInformation()
> >              'prev_page_url' => '/?page=1', 'from' => 3, 'to' => 4, 'data' => ['item3', 'item4'],
> >          ], $p->toArray());
> >      }
> > +
> > +    public function testPaginatorGeneratesUrlsWithoutTrailingSlash()
>
> Should also ensure that it works on the base root too (e.g.
> http://website.com/) as that *should* end in a slash, right? I know very
> few people have pagination on their home page, but something like Tumblr
> would.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/laravel/framework/pull/10186/files#r38929967>.
>